RichText.cpp

```diff
@@ -12,6 +14,39 @@
 namespace sfe
 {
 
+////////////////////////////////////////////////////////////////////////////////
+void RichText::Line::setCharacterColor(std::size_t pos, sf::Color color)
+{
+    assert(pos < getLength());
+    assert(pos >= 0);
+    isolateCharacter(pos);
+    std::size_t stringToFormat = convertLinePosToLocal(pos);
+    m_texts[stringToFormat].setColor(color);
+    updateGeometry();
+}
+
+////////////////////////////////////////////////////////////////////////////////
+void RichText::Line::setCharacterStyle(std::size_t pos, sf::Text::Style style)
+{
+    assert(pos < getLength());
+    assert(pos >= 0);
```

Comment:
Pointless as std::size_t is guaranteed to be unsigned, so it can never be smaller than 0.